ui/packages/consul-ui/tests/acceptance/dc/services/show-routing.feature

```diff
@@ -52,6 +52,7 @@ Feature: dc / services / show-routing: Show Routing for Service
       service: service-1
     ---
     And I see routing on the tabs
+    And I wait a tick
```

Comment:
Really these kind of implementation details shouldn't be in the tests themselves

Apart from that FYI: We have a `And pause for $milliseconds` > `And pause for 0`:

https://github.com/hashicorp/consul/blob/095764a482afb9bfa90f619a2937c7d93112dcd4/ui/packages/consul-ui/tests/steps/debug/index.js#L12-L16

Which is in the `debug` folder, as it should only really be used for debugging (unless we specifically want to model behaviour where we wait for so many milliseconds), so this is just FYI, overall we shouldn't have to tell the tests they need to wait for a tick here.

Reply:
The tests break without this tick. I'm not sure what's going on and there is nothing wrong with the behavior itself so I added this in.

Reply:
Yeah, we could probably look into this test again in the future and rewrite it without ember-cli-page-object. For now, I think we should be fine for getting ember-qunit v5 landed and then tackling ember-cli-page-object as a follow-up.